features/roster.feature

```diff
@@ -65,13 +65,36 @@ Feature: Roster
 
   Scenario: successfully email roster
     When I press the roster bottom for project of id "1"
-    When I fill in "Email" with "testing1@berkeley.edu"
+    When I fill in "Email" with "daniellee0228@berkeley.edu"
```

Comment:
Every-time we run the cucumber tests you would receive an email, are you sure you want to live in this life style.